bot.ts

```diff
@@ -42,8 +43,22 @@ interface v1SearchResult {
 		rel: string;
 		uri: string;
 	}[];
+}
+
+const TIMEOUT_LIMIT_MS = 2500;
+
+const headers = {
+	"Content-Type": "application/json",
+	"User-Agent": "speedruncom-discord-bot/v3",
```

Comment:
```suggestion
	"User-Agent": "CommunityFinder/v3",
```
It should probably be this. Since "CommunityFinder" is what it's called officially